app/blog/utils.py

```diff
@@ -87,3 +87,18 @@ def answer_question(
         model=model,
     )
     return response["choices"][0]["text"].strip()
+
+
+from django.core.exceptions import ValidationError
+import re
+
+
+def snippet_validator(value):
```

Comment:
If you compile the ragex it's faster for subsequent calls to snippet_validator

The comment about rasing a validation error is redundant because it's obvious that is happening.

```py
def snippet_validator(value):
    max_length = 400
    link_media_regex = re.compile(r"<a.*?/a>|<img.*?/img>|<video.*?/video>|<audio.*?/audio>", flags=re.IGNORECASE)
    value_without_links_media = link_media_regex.sub("", value)
    if len(value_without_links_media) > max_length:
        raise ValidationError(
            f"The snippet cannot have more than {max_length} characters (excluding links and media)."
        )
 ```